manifests/init.pp

```diff
@@ -119,16 +117,4 @@
   }
 
   file { "${ldap::params::prefix}/${ldap::params::config}": content => template('ldap/ldap.conf.erb'), }
```

Comment:
In the case of sssd, doesn't this configuration live in sssd.conf so this should actually belong in nslcd?